playbooks/provision_openstack_instance.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: CVP 1164
+  hosts: '{{ hosts | default("all") }}'
+  become: '{{ become | default("yes") }}'
+  gather_facts: '{{ gather_facts | default("no") }}'
+
+  tasks:
+
+    - name: provision a key pair
+      os_keypair:
+        state: present
+        name: ansible_key
+      register: keypair_output
+
+    - debug:
+        var: keypair_output
+
+    - name: copy the key_pair output to a file
+      copy:
+        content: keypair_output["private_key"]
+        dest: "~/.ssh/keypair_output.pem"
+
+    - name: Create a new OpenShift instance on Virtual Machine
+      os_server:
+        state: present
+        name: vm1
```

Comment:
Can you make the name fetched from extra-vars? 
it would be easier for users to customize without actually changing the playbook.